.github/workflows/integration.yml

```diff
@@ -6,7 +6,9 @@ jobs:
   test:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
+      - name: Install docker-compose
+        run: sudo apt-get install -y docker-compose
```

Comment:
This didn't really work, and the issue is intermittent. Let's leave it out of this PR and see if a simple retry gets us past the issue for now

Reply:
I was trying to fix it fast, but right, dropping it for now, am not sure missing docker-compose is gonna be fixed by retrying :-D